ruka/velero/volumesnapshots/controller/rbac-snapshot-controller.yaml

```diff
@@ -0,0 +1,86 @@
+# RBAC file for the snapshot controller.
```

Comment:
Aren't these resources part of the official velero helm chart? https://artifacthub.io/packages/helm/vmware-tanzu/velero